docs/reference/content/getting-started/installation-guide.md

```diff
@@ -0,0 +1,20 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Installation Guide"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Installation Guide"
+  weight = 1
+  pre = "<i class='fa'></i>"
++++
+
+# Installation
+
+The recommended way to get started using one of the drivers in your project is with a dependency management system.
```

Comment:
Does it make sense to say "one of the drivers" in this context?  There's just the Scala driver..